code/modules/goals/station_goals/bluespace_tap.dm

```diff
@@ -175,6 +173,22 @@
 		/obj/item/pizzabox,
 	)
 
+/obj/effect/spawner/lootdrop/bluespace_tap/mats
+	name = "materials"
+	lootcount = 70 //average miners should get the total of this
+	loot = list(
+		/obj/item/stack/ore/iron = 20,
+		/obj/item/stack/ore/glass = 20,
+		/obj/item/stack/ore/titanium = 5,
+		/obj/item/stack/ore/uranium = 5,
+		/obj/item/stack/ore/diamond = 3,
+		/obj/item/stack/ore/bluespace_crystal = 2,
+		/obj/item/stack/ore/bananium = 0, //as rare as possible
```

Comment:
weight of 0 means it'll never be selected, i'm not sure off hand if decimal weights work, but i'm fairly confident that they don't
you'll want to change every weight up and leave bananium at 1 to reduce the chance of bananium being selected

Reply:
either that or remove it outright considering how rare bananium is supposed to be

Reply:
they do get selected yes its just rare ish

Reply:
but sure